Packages crud - some update

minhkoy opened the pull request.

Commit:
Update:

- Locale
- Helper function
- Update model package

Commit:
Hotfix typescrip error

Commit:
Update:

- Add new api
- Add new query
- Update screen

Commit:
- Cập nhật query single package
- Thay đổi route package detail
- Một số hàm helper liên quan

Commit:
thêm file chưa commit

Commit:
Merge branch 'main' of https://github.com/kurumeii/EcoFarm_FE into packages-crud

Commit:
add new cspell words

Commit:
Merge branch 'main' of https://github.com/kurumeii/EcoFarm_FE into packages-crud

Commit:
- Update EnterpriseDTO, PackageDTO giống server
- Một chút cho PackageDetail screen

Commit:
update cspell

Commit:
- Cơ bản màn package detail
- Merge branch 'main' of https://github.com/kurumeii/EcoFarm_FE into packages-crud

Commit:
- Xử lý reviews (chưa hoàn thành)
- Skeleton màn list sản phẩm (products)

Commit:
Merge branch 'main' of https://github.com/kurumeii/EcoFarm_FE into packages-crud

hoanhxlyn commented on src/config/cspell/en/words.txt:
@minhkoy di chuyển các từ tiếng việt sang folder vi

hoanhxlyn commented on src/config/reducers/packages/index.ts:
@minhkoy  sao lại để limit là 1

minhkoy replied:
test thôi a. E quên ch fix lại về 10
Server nếu k nhận dc ttin limit thì cũng auto để là 10

Commit:
Merge branch 'main' of https://github.com/kurumeii/EcoFarm_FE into packages-crud

Commit:
Merge branch 'packages-crud' of https://github.com/kurumeii/EcoFarm_FE into packages-crud

Commit:
- Skeleton tạo mới package
- Minor fixes
- Chuyển ngữ màn package list -> vi
- Merge branch 'main' of https://github.com/kurumeii/EcoFarm_FE into packages-crud
